webui/tests/cypress/integration/02-loginout.spec.ts

```diff
@@ -25,7 +25,10 @@ describe('Sign in/out', () => {
     cy.visit('/ui');
     cy.get('#avatar').click();
     cy.get(`nav a[href="${LOGOUT_ROUTE}"]`).should('have.lengthOf', 1);
+    cy.server();
+    cy.route('POST', '/logout').as('logoutRequest');
```

Comment:
question: does `cy.route` double up on sending the `/logout` POST request which the `Sign Out` button performs as well? If that's the case, if for whatever reason the signout button fails to send a `/logout` request down the line, will this test pass then?

Reply:
nm, cypress docs were not clear, but found the answer on the good ole' stackoverflow. So it looks like it's a xhr listener essentially and the wait command after the click will wait until it sees the logout POST.

Reply:
https://docs.cypress.io/api/commands/as.html#DOM-element is the doc I went off of